### PR TITLE
Check status for dpkg package

### PR DESCRIPTION
**Pull request by rahul2393**
[Issue](https://github.com/aquasecurity/fanal/issues/126)

### CODE REVIEW

File: analyzer/pkg/dpkg/testdata/dpkg [hunk 1]
```diff
@@ -2130,4 +2130,26 @@ Description: Micro string library: shared library
  This package contains the shared library for ustr.
 Homepage: http://www.and.org/ustr/
 
+Package: git
+Status: uninstall ok installed
```
Comment (knqyf263) [inline analyzer/pkg/dpkg/testdata/dpkg:2134]: I prefer using an actual value. Did you see this value in `Status`? AFAIK, it's like `Status: deinstall ok config-files`.

File: analyzer/pkg/dpkg/dpkg.go [hunk 2]
```diff
@@ -105,14 +106,16 @@ func (a debianPkgAnalyzer) parseDpkgPkg(scanner *bufio.Scanner) (pkg *types.Pack
 			}
 		} else if strings.HasPrefix(line, "Version: ") {
 			version = strings.TrimPrefix(line, "Version: ")
+		} else if strings.HasPrefix(line, "Status: ") {
+			isInstalled = strings.HasPrefix(strings.TrimPrefix(line, "Status: "), "install")
```
Comment (knqyf263) [inline analyzer/pkg/dpkg/dpkg.go:110]: Is the order of statuses always the same? I'm worried that the package is not detected when `Status` is like `ok install installed` In that case, we cannot detect any vulnerability of the package. The best way is we'll check the specification. If we can't find it, we should check with Debian 8, 9, 10 and Ubuntu 16.04, 18.04, 20.04.

File: analyzer/pkg/dpkg/testdata/dpkg (hunk 1 above)
Comment (rahul2393) [reply analyzer/pkg/dpkg/testdata/dpkg:2134]: @knqyf263 fixed

File: analyzer/pkg/dpkg/dpkg.go (hunk 2 above)
Comment (rahul2393) [reply analyzer/pkg/dpkg/dpkg.go]: @knqyf263 Updated

**Pull request merged**